[DOCS-9725] Cloudcraft in Datadog

aliciascott opened the pull request:
<!-- *Note: Please remember to review the Datadog Documentation [Contribution Guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) if you have not yet done so.* -->

### What does this PR do? What is the motivation?
<!-- A brief description of the change being made with this pull request. What is your motivation for the PR? -->

### Merge instructions

<!-- 
If you're waiting for a release or there are other considerations that you want us to be aware of, list them here. 
If the PR is ready to be merged once it receives the required reviews, check the box below after you've created the PR.
-->

Merge readiness:
- [ ] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### Additional notes
<!-- Anything else we should know when reviewing?-->

<!-- Previewing the PR: Assuming you are a Datadog employee and named your branch `<yourname>/<description>`, a preview build will run and links to the preview output will be auto-generated and posted in the PR comments. The links will 404 until the preview build is finished running. -->

srividya-rajagopalan reviewed (changes requested):
My review feedback:
- The explanation of standalone Cloudcraft could be clearer. Consider rephrasing to make it more concise and direct.
- Group-bys and filters should be explained before presets, as presets reference group-by functionality.
- The image associated with the infrastructure diagram is incorrect—it should have the infrastructure layer selected.
- Need to clarify “Filters” vs. “Search and Highlight”
    * Filter: Re-renders the diagram to show only infrastructure that matches the filter criteria.
    * Search and Highlight: Highlights the search criteria in the diagram - does not create a new diagram but the elements that match the search criteria are highlighted and the rest are greyed.
* Security Overlay:
    * By default, it displays Critical, High, and Medium severity issues.
    * Instead of “can be configured by hovering over”, say “can be filtered.”
* Agent Overlay
    * The red dot color will be changed to a greyish tone— confirm this before release.

jamesponddotco commented:
@srividya-rajagopalan asked me to take a look, so here I'm. The documentation looks excelent, and Sri already pointed out a few issues, so I don't have much to add.

> If you have advanced security requirements, it is possible to create a [custom IAM role with more limited permissions](https://docs.datadoghq.com/cloudcraft/advanced/minimal-iam-policy/), however, this may not be compatible with other Datadog products.

The minimal IAM policy from standalone Cloudcraft won't work for Cloudcraft in Datadog, so this portion should be removed.

> Additionally, you can click into a specific security group to see the inbound and outbound rules for that group.

That is true for standalone Cloudcraft, but not for Cloudcraft in Datadog. The connections are displayed in the diagram, though.

----

I also think we may need to update the menu item for standalone Cloudcraft from "Cloudcraft" to "Standalone Cloudcraft", otherwise we have "Cloudcraft" twice in the menu, which may lead to confusion.

aliciascott commented:
Hi @srividya-rajagopalan and @jamesponddotco ! thank you both for your detailed review! I have addressed all of your comments except for the Agent overlay - Making a note here to review prior to release for Agent color and updated screenshot. Can you both please re-review when you get a chance? Thanks!

jamesponddotco commented:
@aliciascott Looks good to me, but I think @srividya-rajagopalan needs to approve it, since she was the requested reviewer.

jhgilbert commented on content/en/datadog_cloudcraft/_index.md:
```diff
@@ -0,0 +1,158 @@
+
+Cloudcraft offers a powerful, live read-only visualization tool for cloud architecture, enabling you to explore, analyze, and manage your infrastructure with ease. Not to be confused with the [Standalone Cloudcraft documentation][1], this guide outlines the functionality, setup, and use cases of Cloudcraft *in Datadog*, detailing its benefits for various user personas, and highlighting key featur ...
+
+<div class="alert alert-info">This documentation applies to the Cloudcraft <em>in Datadog</em> product. For information on the standalone Cloudcraft product, please refer to the <a href="/cloudcraft"> Cloudcraft (Standalone) </a> documentation.</div>
```
```suggestion
<div class="alert alert-info">This documentation applies to the Cloudcraft <em>in Datadog</em> product. For information on the standalone Cloudcraft product, please refer to the <a href="/cloudcraft">Cloudcraft (Standalone)</a> documentation.</div>
```
Some of this extra space is showing up in the link, which looks a bit odd, so I just removed any unnecessary spaces here

jhgilbert commented on content/en/datadog_cloudcraft/_index.md:
```diff
@@ -0,0 +1,158 @@
+### Prerequisites
+
+- [Resource collection][2] must be enabled for your AWS accounts.
+- For the best experience, Datadog strongly recommends to use the AWS-managed [`SecurityAudit`][5] policy, or the more permissive [`ReadOnlyAccess`][6] policy.
```
```suggestion
- For the best experience, Datadog strongly recommends using the AWS-managed [`SecurityAudit`][5] policy, or the more permissive [`ReadOnlyAccess`][6] policy.
```

jhgilbert commented on content/en/datadog_cloudcraft/_index.md:
```diff
@@ -0,0 +1,158 @@
+- For the best experience, Datadog strongly recommends to use the AWS-managed [`SecurityAudit`][5] policy, or the more permissive [`ReadOnlyAccess`][6] policy.
+- To view security misconfigurations on the [Security findings overlay](#security-findings), [CSM][3] must be enabled.
+
+**Note**: Cloudcraft adapts to restrictive permissions by excluding inaccessible resources. For example, if you opt to not grant permission to list S3 buckets, the diagram will simply exclude those buckets. If permissions block certain resources, an alert is displayed in the user-interface(UI).
```
```suggestion
**Note**: Cloudcraft adapts to restrictive permissions by excluding inaccessible resources. For example, if you opt to not grant permission to list S3 buckets, the diagram will simply exclude those buckets. If permissions block certain resources, an alert is displayed in the user interface (UI).
```

jhgilbert replied:
I think UI is such a common acronym at this point that you could probably use "UI" throughout with no issue, but I leave it to you!

jhgilbert commented on content/en/datadog_cloudcraft/_index.md:
```diff
@@ -0,0 +1,158 @@
+
+**Note**: The account name in the **Account** dropdown originates from your AWS account tags in the AWS integration tile.
+
+## Group by
```
```suggestion
## Group By
```
For consistency with the next line

jhgilbert commented on content/en/datadog_cloudcraft/_index.md:
```diff
@@ -0,0 +1,158 @@
+
+## Getting started
+
+To get started using Cloudcraft, select one or more accounts, regions, and resources. The diagram automatically displays the Infrastructure diagram.
```
I recommend including a link to the page in Datadog they're supposed to start from, since it's not clear where this action should happen.

jhgilbert commented on content/en/datadog_cloudcraft/_index.md:
```diff
@@ -0,0 +1,158 @@
+
+### Infrastructure diagram
+
+The infrastructure view provides a broad overview, grouping resources by Account, Region and VPC. This view is ideal for generating architecture diagrams for troubleshooting or high-level review.
```
```suggestion
The infrastructure view provides a broad overview, grouping resources by Account, Region, and VPC. This view is ideal for generating architecture diagrams for troubleshooting or high-level review.
```

jhgilbert commented on content/en/datadog_cloudcraft/_index.md:
```diff
@@ -0,0 +1,158 @@
+
+### Saved views 
+
+To apply a saved view to your diagram:
```
This is the first mention of saved views, just wanted to verify that the customer is expected to already understand what these are?

aliciascott replied:
thanks great callout ! I've added more copy here

jhgilbert commented on content/en/datadog_cloudcraft/_index.md:
```diff
@@ -0,0 +1,158 @@
+
+## Explore resources
+
+In any of the Cloudcraft presets, you utilize the zoom and hover features to pinpoint the most critical resources. As you zoom in, additional resource names become visible. Hovering over a resource reveals a hover panel with basic information. Clicking on a resource opens a side panel that displays its Datadog observability, cost information, and security data, with cross-links to other Datadog p ...
```
```suggestion
In any of the Cloudcraft presets, use the zoom and hover features to pinpoint the most critical resources. As you zoom in, additional resource names become visible. Hovering over a resource reveals a hover panel with basic information. Clicking on a resource opens a side panel that displays its Datadog observability, cost information, and security data, with cross-links to other Datadog products where relevant.
```
Or "you can use"

jhgilbert commented on content/en/datadog_cloudcraft/_index.md:
```diff
@@ -0,0 +1,158 @@
+The security findings overlay in Cloudcraft provides an overlay from CSM misconfigurations, allowing you to quickly identify CSM findings. This allows you to:
+
+- Identify security issues in infrastructure diagrams.  
+- View misconfigurations in context to analyze their impact and prioritize remediation  
```
```suggestion
- View misconfigurations in context to analyze their impact and prioritize remediation.
```

jhgilbert commented on content/en/datadog_cloudcraft/_index.md:
```diff
@@ -0,0 +1,158 @@
+
+### Agent Overlay
+
+The Agent overlay indicates whether the Agent is installed on your EC2 hosts using a collapsible legend. A green dot signifies the Agent is installed, while a red dot indicates it is not installed on that resource.
```
```suggestion
The Agent Overlay indicates whether the Agent is installed on your EC2 hosts using a collapsible legend. A green dot signifies the Agent is installed, while a red dot indicates it is not installed on that resource.
```
Just for consistency with the header, you could also lowercase both

jhgilbert commented on content/en/datadog_cloudcraft/_index.md:
```diff
@@ -0,0 +1,158 @@
+## FAQ 
+
+Why are there two Cloudcraft's and which one is right for me ?
+Add Copy here
```
I know you're waiting for this, just let me know when it's ready for review!

aliciascott commented:
/merge
